### PR TITLE
opae-sdk: change  phy info OPAE  ERROR output  to MSG

### PR DESCRIPTION
**Pull request by anandaravuri**
Signed-off-by: anandaravuri <ananda.ravuri@intel.com>

### CODE REVIEW

File: libraries/libboard/board_n6000/board_n6000.c [hunk 1]
```diff
@@ -512,7 +512,7 @@ fpga_result print_phy_info(fpga_token token)
 
 	res = find_dev_feature(token, HSSI_FEATURE_ID, feature_dev);
 	if (res != FPGA_OK) {
-		OPAE_ERR("Failed to find feature ");
+		OPAE_MSG("Failed to find feature HSSI");
```
Comment (tswhison) [inline libraries/libboard/board_n6000/board_n6000.c:515]: OPAE_MSG or OPAE_DBG?

File: libraries/libboard/board_n6000/board_n6000.c [hunk 2]
```diff
@@ -549,7 +549,7 @@ fpga_result print_sec_info(fpga_token token)
 	res = fpgaTokenGetObject(token, DFL_SEC_PMCI_GLOB, &tcm_object,
 		FPGA_OBJECT_GLOB);
 	if (res != FPGA_OK) {
-		OPAE_ERR("Failed to get token Object");
+		OPAE_MSG("Failed to get token Object");
```
Comment (tswhison) [inline libraries/libboard/board_n6000/board_n6000.c:552]: OPAE_MSG or OPAE_DBG?

File: libraries/libboard/board_n6000/board_n6000.c (hunk 1 above)
Comment (anandaravuri) [reply libraries/libboard/board_n6000/board_n6000.c:515]: OPAE_MSG

File: libraries/libboard/board_n6000/board_n6000.c (hunk 2 above)
Comment (anandaravuri) [reply libraries/libboard/board_n6000/board_n6000.c:552]: OPAE_MSG

**Review by tswhison — APPROVED** [review]

**Pull request merged**